pkg/apis/kubermatic/v1/alertmanager.go

```diff
@@ -31,12 +31,14 @@ const (
 
 // +kubebuilder:object:generate=true
 // +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
 
 type Alertmanager struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec AlertmanagerSpec `json:"spec,omitempty"`
+	Spec   AlertmanagerSpec   `json:"spec,omitempty"`
+	Status AlertManagerStatus `json:"status,omitempty"`
```

Comment:
Can we please settle on "Alertmanager" _or_ "AlertManager"? Prefaribly the first, because it's t he official spelling.

Reply:
done

Reply:
And I've also added missing clonner for AlertmanagerStatus